public/app/features/alerting/unified/state/actions.ts

```diff
@@ -114,27 +114,49 @@ export const fetchAlertManagerConfigAction = createAsyncThunk(
           }));
         }
 
+        const { data: amFeatures } = await thunkAPI.dispatch(
+          featureDiscoveryApi.endpoints.discoverAmFeatures.initiate({
+            amSourceName: alertManagerSourceName,
+          })
+        );
+
+        const lazyConfigInitSupported = amFeatures?.lazyConfigInit ?? false;
```

Comment:
Can we add a comment here what this `lazyConfigInit` means?

Reply:
Good point!